specs/PointOfInterest/Museum/doc/spec.md

```diff
@@ -95,23 +98,21 @@ used by applications.
 -   `historicalPeriod` : Corresponds to the historical period(s) of the
     exhibitions made by this museum.
     -   Attribute type: List of [Text](https://schema.org/Text)
-    -   Allowed values:
-        -   A ISO8601 time interval. For example `1920/1940`. The second element
-            of the interval can be left empty to denote "till now".
-        -   A comma separated list of years, for instance `1620,1625,1718`.
-        -   A century, represented by a year pattern, for instance `19xx` would
-            correspond to the twentieth century. And `196x` would correspond to
-            the sixties decade.
+    -   Allowed values: - A ISO8601 time interval. For example `1920/1940`. The
```

Comment:
Allowed values such as "A ISO8601 time interval and A Century should be bullets

Actually being picky it should be **An** ISO8601 ...